### PR TITLE
#477 raw text

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
add raw boolean property to text 
When true, use a simple span to render (no default style)

### CODE REVIEW

**Review by dinhlongviolin1 — APPROVED** [review]

File: gui/doc/viselements.json [hunk 1]
```diff
@@ -15,6 +15,12 @@
             "default_value": "\"\"",
             "doc": "The value displayed as text by this control."
           },
+          {
+            "name": "raw",
+            "type": "bool",
+            "default_value": "False",
+            "doc": "If set to True, the component renders as an HTML SPAN element without any default style"
```
Comment (FabienLelaquais) [inline gui/doc/viselements.json]: Ending dot missing.
SPAN -> <span> ?

File: src/taipy/gui/renderers/factory.py [hunk 2]
```diff
@@ -461,11 +461,7 @@ class _Factory:
         .set_value_and_default(with_update=False)
         ._set_dataType()
         .set_attributes(
-            [
-                ("format",),
-                ("id",),
-                ("hover_text", PropertyType.dynamic_string),
-            ]
+            [("format",), ("id",), ("hover_text", PropertyType.dynamic_string), ("raw", PropertyType.boolean, False)]
```
Comment (FabienLelaquais) [inline src/taipy/gui/renderers/factory.py:464]: Dynamic? Really??

**Review by FabienLelaquais — APPROVED** [review]
Minor comments.

File: gui/doc/viselements.json (hunk 1 above)
Comment (FredLL-Avaiga) [reply gui/doc/viselements.json]: ? -> SPAN ?

File: src/taipy/gui/renderers/factory.py (hunk 2 above)
Comment (FredLL-Avaiga) [reply src/taipy/gui/renderers/factory.py:464]: yes

File: gui/doc/viselements.json (hunk 1 above)
Comment (FredLL-Avaiga) [reply gui/doc/viselements.json]: ok &lt;span&gt;

**Pull request merged**